optimizely/helpers/enums.py

```diff
@@ -157,3 +157,9 @@ class NotificationTypes(object):
     OPTIMIZELY_CONFIG_UPDATE = 'OPTIMIZELY_CONFIG_UPDATE'
     TRACK = 'TRACK:event_key, user_id, attributes, event_tags, event'
     LOG_EVENT = 'LOG_EVENT:log_event'
+
+
+class VersionType(object):
+    IS_PRE_RELEASE = '-'
+    HAS_WHITE_SPACE = ' '
+    IS_BUILD = '+'
```

Comment:
This feels confusing. `HAS_WHITE_SPACE` denotes invalid version and does not belong here. It should not be in this list and be checked separately.